platform/ios/src/MGLMapView.h

```diff
@@ -753,7 +753,8 @@ MGL_EXPORT
  @param coordinate The new center coordinate for the map.
  @param animated Specify `YES` if you want the map view to scroll to the new
     location or `NO` if you want the map to display the new location
-    immediately.
+    immediately. If using `YES`, wait until the map view has finished loading
```

Comment:
`@param` definitions are usually best kept concise — a single sentence description of the purpose of that parameter. For usage considerations, the description body is a more appropriate place.

> For animated changes, wait until...